security/guard_authentication.rst

```diff
@@ -505,7 +505,7 @@ and add the following logic::
             $csrfToken = $request->request->get('_csrf_token');
             
             if (false === $this->csrfTokenManager->isTokenValid(new CsrfToken('authenticate', $csrfToken))) {
-                throw new InvalidCsrfTokenException('Invalid CSRF token.');
+                return null;
```

Comment:
Hi, @HeahDude ,

Okay, I see it now. Nevertheless, if you throw an exception the application stays in a redirect loop until the browser kills it. Anyway, I will look further onto it in September, to try to solve the problem directly.

Thank you!

Regards.